### DISCUSSION TITLE
Applying overly effects to outputted images

### DISCUSSION THREAD

**Discussion by franciscoEdwards** [joonaspaakko/Batch-Mockup-Smart-Object-Replacement-photoshop-script#25] [Q&A] [answered]
Hello there! 

Thanks for coding this Photoshop script, it's fantastic! I got it working on the Mac without any problems.

When I used it on my project, I noticed that some effects (gradients and texturizer overlay, etc.) that were on the smart object were not applied to the outputted images. Is there anything that I can do to fix this issue? Or did I miss something in the documentation about this?

Any help is appreciated.

Thanks!

Frank E.

**Comment by joonaspaakko** [comment, marked as answer]
Is your Photoshop older than PS CC? It sounds similar to  https://github.com/joonaspaakko/Batch-Mockup-Smart-Object-Replacement-photoshop-script/discussions/15#discussion-3936694

If you can send me the project files, I'll take a look at it. My email is in my github profile, if needed. It doesn't have to be the same exact files, as long as it can reproduce the issue with it.

**Reply by franciscoEdwards** [reply]
Got it. I see my mistake now. 

I was using Photoshop CC (Beta) and that was causing the issue. Running the standard version of Photoshop CC resolved the issue. Sorry, I missed the previous comment on this issue. Thank you for the assistance.

**Reply by joonaspaakko** [reply]
Question; was this beta version super old or a recent beta version? If it's a recent beta version and you downgraded to stable release, it could mean it might not work so well in the future...

**Reply by franciscoEdwards** [reply]
I was using the most recent beta version (Version: 24.7.0) when I encounter the issue. Downgrading to the stable release (Version: 24.6.0) fixed the issue. I hope this issue doesn't persist in the future.

**Reply by joonaspaakko** [reply]
Thanks for letting me know.
